### ISSUE TITLE
Enforce wrapped parameter styling.

### ISSUE THREAD

**Issue by JakeWharton** [shyiko/ktlint#130]
Both the JetBrains and Android style guide require
```kotlin
fun foo(
    bar: Bar,
    baz: Baz
) {
```
when the signature does not fit on one line.

Currently
```kotlin
fun foo(bar: Bar
        baz: Baz) {
```
is incorrectly marked valid (#26) which is not only incorrect but also encourages the bad practice of horizontally aligning things.

**Comment by shyiko** [comment]
Thank you for the ticket. I'll make sure this is fixed as part of https://github.com/shyiko/ktlint/pull/118.

**Comment by MyDogTom** [comment]
These changes are included into `IndentationRule` and `ClassAndFunctionHeaderFormatRule`. 
It's already possible to try them by using snapshot KtLint version - https://github.com/shyiko/ktlint#development
Alternatively, you can include https://github.com/MyDogTom/ktlint-custom-rules . This is just full copy of the `IndentationRule` and `ClassAndFunctionHeaderFormatRule` from `ktlint:master`.

I'm pretty sure, some cases are not handled, but it's almost impossible to find them without feedback from other people.

**Comment by shyiko** [comment]
Fixed in [0.16.0](https://github.com/shyiko/ktlint/releases/tag/0.16.0).
